### PR TITLE
migration of gce-disk dashboard from legacy to sample repo

### PR DESCRIPTION
**Pull request by philshin-google**
added in metadata and config files for gce-disk dashboard.

### CODE REVIEW

File: dashboards/google-gce-disk/gce-disk.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "displayName": "Disks",
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "width": 48,
+        "height": 35,
+        "widget": {
+          "title": "disk bytes used",
```
Comment (varun-c) [inline dashboards/google-gce-disk/gce-disk.json:10]:
```suggestion
          "title": "Disk bytes used",
```

Should the title be sentence case?

File: dashboards/google-gce-disk/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+sample-dashboards:
+  -
+    category: GCE Disk
+    id: gce-disk
+    display_name: Disks
```
Comment (varun-c) [inline dashboards/google-gce-disk/metadata.yaml:5]: Should we add a description for the dashboard?

**Review by varun-c — APPROVED** [review]

**Review by EvanSimpson — CHANGES_REQUESTED** [review]
Can we also rename gce-disk everywhere to compute-disk?
e.g. google-compute disk as the folder, compute-disk as the file name/ID?

**Comment by philshin-google** [comment]
> Can we also rename gce-disk everywhere to compute-disk? e.g. google-compute disk as the folder, compute-disk as the file name/ID?

done!

File: dashboards/google-compute-disk/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+sample-dashboards:
+  -
+    category: Compute Disk
+    id: compute-disk
+    display_name: Disks
+    description: |-
+    This dashboard has charts displaying an inventory of GCE Disks, as well as Volume throughput and IOPS.
```
Comment (EvanSimpson) [inline dashboards/google-compute-disk/metadata.yaml:7]: I think this needs an additional indent

```suggestion
      This dashboard has charts displaying an inventory of GCE Disks, Volume throughput, and IOPS.
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**